l1-contracts/scripts/register-hyperchain.ts

```diff
@@ -99,11 +100,13 @@ async function main() {
       await checkTokenAddress(baseTokenAddress);
       console.log(`Using base token address: ${baseTokenAddress}`);
 
+      const useGovernance = cmd.useGovernance ? cmd.useGovernance == "true" : false;
```

Comment:
maybe you could do !!cmd.useGovernance ? 
https://basarat.gitbook.io/typescript/recap/truthy

Reply:
Good point - fixed